examples/hello/Dockerfile

```diff
@@ -1,9 +1,10 @@
 # ch-test-scope: standard
-FROM debian:stretch
+# ch-test-builder-exclude: ch-grow  #see issues #472 and #498
```

Comment:
This example needs to still work with `ch-grow`. You could comment that we haven't converted it to CentOS for that reason, though.

Reply:
Sounds good, I will revert.